prisma/schema.prisma

```diff
@@ -21,6 +21,7 @@ model Meeting {
   attendees   Attendee[]
   bubbleId    String?
   started    Boolean    @default(false)
+  status      String     @default("scheduled") // "scheduled", "started", "ended"
```

Comment:
Can we create a prisma enum ? 
https://www.prisma.io/docs/concepts/components/prisma-schema/data-model